data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -69,11 +69,16 @@ spec:
               memory: "15Mi"
           securityContext:
             privileged: true
+            readOnlyRootFilesystem: true
           volumeMounts:
             - name: cnibin
               mountPath: /opt/cni/bin
+            - name: tmp
+              mountPath: /tmp
```

Comment:
why do we need access to /tmp ? Seems we didn't need it before. Why add it now ?

Reply:
From the commit message:
```
Since some system utilities (e.g., bash, cp, sha256sum) may require a
writable /tmp directory, mount an emptyDir at /tmp to provide a writable
scratch space, following container security best practices [0].
Also update the associated SCC to enforce readOnlyRootFilesystem: true
at the policy level.

[0]
https://redhat-best-practices-for-k8s.github.io/guide/#k8s-best-practices-storage:-emptydir
```

Reply:
IOW, if the files are too big they sometimes write to tmp. To be safe - mount an empty tmp folder, so that it won't explode should some cp decides to use that folder..